samples/model-agnositc/model-analysis.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import kfp.dsl as dsl
+import datetime
+
+
+def dataflow_tf_model_analyze_op(image: 'GcrUri', csv_eval_path_with_predict: 'GscUri', schema: 'GcsUri[text/json]', output: 'analysis_output', step_name='analysis'):
+    return mlp.ContainerOp(
+        name = step_name,
+        image = image,
+        arguments = [
+            '--csv_eval_with_predict_path', csv_eval_path_with_predict,
+            '--schema_path', schema,
+            '--output', output,
+        ],
+    )
+
+@dsl.pipeline(
+  name='TFMA Model Agnostic Example',
+  description='Example pipeline that does model analysis based on a csv file with features, predicitons, labels.'
+)
+def model_analysis(
+    output: dsl.PipelineParam,
+    project: dsl.PipelineParam,
+
+    image: mlp.PipelineParam=mlp.PipelineParam(
```

Comment:
Container image as a pipeline parameter is not supported yet. Can you hard code the container image here?

Reply:
@salehay